### PR TITLE
chore: remove more antd

### PR DESCRIPTION
**Pull request by daibhin**
## Changes

|Before|After|
|----|----|
| Screenshot 2024-03-11 at 11 30 27 | Screenshot 2024-03-11 at 16 30 24  |

|Before|After|
|----|----|
| Screenshot 2024-03-11 at 11 07 26 | Screenshot 2024-03-11 at 11 24 17 |

|Before|After|
|----|----|
| before | after |

## How did you test this code?

Replaced every instance manually

### CODE REVIEW

File: frontend/src/lib/lemon-ui/LemonSelect/LemonSelect.tsx [hunk 1]
```diff
@@ -138,6 +138,7 @@ export function LemonSelect<T extends string | number | boolean | null>({
                 .flatMap((i) => (isLemonMenuSection(i) ? i.items.filter(Boolean) : i))
                 .findIndex((i) => (i as LemonMenuItem).active)}
             closeParentPopoverOnClickInside={menu?.closeParentPopoverOnClickInside}
+            closeOnClickInside={false}
```
Comment (daibhin) [inline frontend/src/lib/lemon-ui/LemonSelect/LemonSelect.tsx:141]:
```suggestion
```

File: frontend/src/scenes/plugins/source/PluginSource.tsx [hunk 2]
```diff
@@ -106,9 +105,13 @@ export function PluginSource({
                             .
                         </p>
 
-                        {pluginSourceLoading ? (
-                            // eslint-disable-next-line react/forbid-elements
-                            <Skeleton />
+                        {true ? (
```
Comment (daibhin) [inline frontend/src/scenes/plugins/source/PluginSource.tsx:108]:
```suggestion
                        {pluginSourceLoading ? (
```

File: frontend/src/scenes/settings/organization/VerifiedDomains/SSOSelect.tsx [hunk 3]
```diff
@@ -20,29 +20,36 @@ export function SSOSelect({ value, loading, onChange, samlAvailable }: SSOSelect
         return null
     }
 
+    const authProviders = Object.keys(preflight.available_social_auth_providers) as SSOProvider[]
+
```
Comment (daibhin) [inline frontend/src/scenes/settings/organization/VerifiedDomains/SSOSelect.tsx:24]:
```suggestion
```

**Review by benjackwhite — APPROVED** [review]
Nit otherwise 👍 👍 👍 👍 👍 👍 👍 👍 👍

File: frontend/src/scenes/insights/views/Funnels/FunnelBinsPicker.tsx [hunk 4]
```diff
@@ -33,83 +31,78 @@ const options: BinOption[] = [
...
+    const selectedValue = funnelsFilter?.binCount || BIN_COUNT_AUTO
+    const selectedOption = BIN_OPTIONS.find((o) => o.value === selectedValue)
+
+    const overlay = (
+        <div className="space-y-px" onClick={(e) => e.stopPropagation()}>
+            {preferredOptions.map((bin) => (
+                <LemonButton
+                    fullWidth
+                    key={bin.value}
+                    icon={<IconGraph />}
```
Comment (daibhin) [inline frontend/src/scenes/insights/views/Funnels/FunnelBinsPicker.tsx:63]:
```suggestion
```

**Pull request merged**